### PR TITLE
changed entropy to BioQC version

### PR DESCRIPTION
**Pull request by czackl**
Updated "scoreEntropy" to BioQC::entropySpecificity

### CODE REVIEW

**Review by federicomarini — COMMENTED** [review]
We probably need to also adjust the imports, I'll search for the spot and tag you there @czackl

File: R/SignatureExplorationPlots.R [hunk 1]
```diff
@@ -128,7 +128,7 @@ plot_meanEntropyPerMethod <- function(signature_list,
 
   # calculate Mean Entropy for each signature
   for (name in names(signature_list)) {
-    meanEntropy <- mean(apply(signature_list[[name]], 1, scoreEntropy))
+    meanEntropy <- mean(BioQC::entropySpecificity(signature_list[[name]]))
```
[the remark this answers is not in the document]
Comment (czackl) [reply R/SignatureExplorationPlots.R:131]: Added

File: R/SignatureRefinements.R [hunk 2]
```diff
@@ -188,7 +188,8 @@ selectGenesByScore <- function(signature_mat,
 
     score <- list()
     if (scoring_method == "entropy") {
-      score[gene] <- scoreEntropy(row) # calculate score and save named result
+      # score[gene] <- scoreEntropy(row) # calculate score and save named result
```
[the remark this answers is not in the document]
Comment (czackl) [reply R/SignatureRefinements.R:191]: I guess we will decide on one implementation in this case

[the remark this answers is not in the document]
Comment (federicomarini) [reply R/SignatureRefinements.R:191]: sounds good to me ;)

**Pull request merged**